File: eshop-model/src/main/java/model/Fournisseur.java
package model;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;

@Entity
@DiscriminatorValue("fournisseur")
public class Fournisseur extends Personne {

	
	private String societe;

	public Fournisseur() {
	}

	public Fournisseur(String prenom, String nom, String societe) {
		super(prenom, nom);
		this.societe = societe;
	}

	public String getSociete() {
		return societe;
	}

	public void setSociete(String societe) {
		this.societe = societe;
	}

	@Override
	public String toString() {
		return "Fournisseur [numero=" + numero + ", prenom=" + prenom + ", nom=" + nom + ", societe=" + societe + "]";
	}
	
	
	
	
	
	
	
	
	
	
	
}

File: olivier/angular/formationAngular/src/app/model/fournisseur.ts
import { Personne } from './personne';
export class Fournisseur extends Personne {
  public constructor(prenom?: string, nom?: string, private _societe?: string) {
    super(prenom, nom);
  }

  public get societe(): string | undefined {
    return this._societe;
  }

  public set societe(value: string | undefined) {
    this._societe = value;
  }
}
